### PR TITLE
Change provider name

### PR DESCRIPTION
**Pull request by pkliczewski**
Signed-off-by: Piotr Kliczewski <piotr.kliczewski@gmail.com>

### CODE REVIEW

**Comment by pkliczewski** [comment]
@yaacov FYI

**Comment by yaacov** [comment]
Thanks !

File: build.sh [hunk 1]
```diff
@@ -98,12 +98,12 @@ do_build_conversion() {
     #${CONTAINER_MGMT} push quay.io/nyoxi/kubevirt-conversion:latest
 }
 
-do_build_vmware() {
+do_build_provider() {
     TAG="v$KUBEVIRT_VERSION-$KUBEVIRT_VMWARE_RELEASE"
```
Comment (suomiy) [inline build.sh:102]: can we change this variable as well, or is somebody dependending on it?

File: kubevirt-vmware/build/Dockerfile [hunk 2]
```diff
@@ -1,15 +0,0 @@
-FROM alpine:3.8
```
Comment (suomiy) [inline kubevirt-vmware/build/Dockerfile:1]: should theses build files be deleted? It is hard to build the project without them :)

**Review by suomiy — CHANGES_REQUESTED** [review]

**Comment by suomiy** [comment]
btw, the bug URL is probably off

File: build.sh (hunk 1 above)
Comment (pkliczewski) [reply build.sh:102]: Will update

File: kubevirt-vmware/build/Dockerfile (hunk 2 above)
Comment (pkliczewski) [reply kubevirt-vmware/build/Dockerfile:1]: This file was moved to new directory. I agree that it is needed to build the image.

**Comment by pkliczewski** [comment]
@suomiy please take a look

**Comment by suomiy** [comment]
@pkliczewski 

I am still getting:

```bash
$ ./build.sh  build_provider
FATA[0000] must run command in project root dir: project structure requires build/Dockerfile
```

**Comment by pkliczewski** [comment]
@suomiy I think you need to clean build folders. here is my output:

> 12:27 $ ./build.sh build_provider
> INFO[0000] Building OCI image quay.io/kubevirt/vm-import-provider:v2.0.0-9 
> Sending build context to Docker daemon  107.6MB
> Step 1/7 : FROM alpine:3.8
> 3.8: Pulling from library/alpine
> 486039affc0a: Pull complete 
> Digest: sha256:2bb501e6173d9d006e56de5bce2720eb06396803300fe1687b58a7ff32bf4c14
> Status: Downloaded newer image for alpine:3.8
> ---> c8bccc0af957
> Step 2/7 : ENV OPERATOR=/usr/local/bin/vm-import-provider     USER_UID=1001     USER_NAME=vm-import-provider
> ....

**Comment by suomiy** [comment]
I don't know if I am missing something, but this still happens on clear clone

**Comment by suomiy** [comment]
```bash
$ git clone https://github.com/pkliczewski/manageiq-v2v-conversion_host.git
Cloning into 'manageiq-v2v-conversion_host'...
$ cd manageiq-v2v-conversion_host
$ ./build.sh  build_provider
FATA[0000] must run command in project root dir: project structure requires build/Dockerfile 
```

**Comment by pkliczewski** [comment]
That is interesting. It seems like stale build cache was on my side. Thanks!

File: vm-import-provider/build/bin/entrypoint [hunk 3]
```diff
@@ -5,7 +5,7 @@
 
 if ! whoami &>/dev/null; then
   if [ -w /etc/passwd ]; then
-    echo "${USER_NAME:-kubevirt-vmware}:x:$(id -u):$(id -g):${USER_NAME:-kubevirt-vmware} user:${HOME}:/sbin/nologin" >> /etc/passwd
+    echo "${USER_NAME:-vm-import-provider}:x:$(id -u):$(id -g):${USER_NAME:-kubevirt-vmware} user:${HOME}:/sbin/nologin" >> /etc/passwd
```
Comment (suomiy) [inline vm-import-provider/build/bin/entrypoint:8]: nit

```suggestion
    echo "${USER_NAME:-vm-import-provider}:x:$(id -u):$(id -g):${USER_NAME:-vm-import-provider} user:${HOME}:/sbin/nologin" >> /etc/passwd
```

**Comment by suomiy** [comment]
now it build fine :)

File: vm-import-provider/build/bin/entrypoint (hunk 3 above)
Comment (pkliczewski) [reply vm-import-provider/build/bin/entrypoint]: :+1:

**Pull request merged**